Documentation/ApiOverview/CropVariants/General/Index.rst

```diff
@@ -0,0 +1,139 @@
+.. include:: ../../Includes.txt
+
+.. _cropvariants:
+
+=====================
+General Configuration
+=====================
+
+The following examples are basically meant to add one single
+cropping configuration to sys_file_reference,  which will then apply in every
+record, which reference images.
+
+In this example we configure two crop variants, one with the id "mobile",
+one with the id "desktop". The array key defines the crop variant id, which will be used
+when rendering an image with the image view helper.
+
+The allowed crop areas are now also configured differently.
```

Comment:
"now" as to compared with when and what?

Reply:
"now" as compared to versions prior 8 LTS - removed it completely